core/drivers/remoteproc/stm32_remoteproc.c

```diff
@@ -381,6 +384,16 @@ static TEE_Result stm32_rproc_probe(const void *fdt, int node,
 			goto err;
 	}
 
+#ifdef CFG_STM32MP15
+	if (!stm32_rcc_is_secure()) {
+		IMSG("WARNING: insecure rproc isolation: RCC tzen:0");
```

Comment:
a bit too much of ":", proposition:
`WARNING: insecure rproc isolation, RCC tzen:0`